v2/index.html

```diff
@@ -247,16 +247,27 @@ <h3>Key Signature</h3>
         <h3>Time Signature</h3>
         <div class="issue" data-number="35"></div>
         <p>
-            The time signature is preceded by <code>@</code> and indicates the time value or the mensuration sign of the
-            incipit. If the incipit has no time signature, the <code>@</code> is omitted.
+            A time signature or mensuration sign SHOULD be provided, as required by the repertoire and source. It MAY
+            be omitted if appropriate.
+        </p>
+        <p>
+            There are two main categories of time signature forms, "Common Western Music Notation" (CWMN) and
+            "Mensural". For CWMN time signatures expressed as one number above another, transcribe these numbers
+            separated by a <code>/</code> character. For "common" and "alla breve" key signatures, use <code>c</code>
+            and <code>c/</code>, respectively.
+        </p>
+        <p>
+            For mensuration signs, the <code>c</code> and <code>o</code> characters indicate <em>tempus
+            imperfectus</em> and <em>tempus perfectus</em>, respectively. The <code>.</code> character indicates
+            "major" prolation; no <code>.</code> indicates "minor" prolation. Numbers following these signs
+            indicate <em>Modus cum tempore</em> signs with a numeral. A <code>/</code> character may follow the
+            <em>tempus</em> character to indicate diminution.
         </p>
-        <div class="issue" data-number="37"></div>
-        <div class="issue" data-number="35"></div>
         <p>
-            Fractional or numeric values are transcribed as fractions and mensuration signs are transcribed with a
-            lowercase letter, if necessary followed by <code>/</code> or <code>.</code>:
+            CWMN time signatures that indicate alternating measures MAY be provided by transcribing both. These MUST
```

Comment:
The is not problem with the description here. However, I wonder if we could avoid in PAE2 the use of a space. Ideally, a space should indicate the end of the time signature. In fact, I think we have a real problem if we want to forbid spaces in one-line incipit input, where the score definition is delimited by a space.

A possible character we could use insteas is `-`. Any other ideas?

Reply:
The pipe `|`, although I am conscious that some keyboard layouts might not have this readily accessible?

Reply:
We can create a new issue to track this change so that it doesn't hold up this PR